### DISCUSSION TITLE
Welcome to sauron Discussions!

### DISCUSSION THREAD

**Discussion by ivanceras** [ivanceras/sauron#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fungos** [comment]
Hello, I'm starting into web frontend with rust, and I've chosen Sauron for my experiments. After a few days messing around, I wonder if there is a live chat available (elsewhere other than Github) to iteract with other users.

**Comment by ivanceras** [comment]
I've just started a discord channel here: https://discord.gg/93n6rchh

**Reply by Benjamin-Dagarin** [reply]
Is this discord channel still active? Invitation link has expired and I would like to join.
